src/convertToObject.js

```diff
@@ -7,6 +7,26 @@
  */
 function convertToObject(sourceString) {
   // write your code here
+  // eslint-disable-next-line no-debugger
+  if (!sourceString) {
+    return {};
+  }
+
+  const styleToObject = {};
+  const partsOfExpresions = sourceString.split(';');
```

Comment:
The variable name 'partsOfExpresions' contains a typo and is not descriptive enough. Consider renaming it to 'styleDeclarations' to more accurately reflect its contents.